distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -28,7 +28,7 @@
 # org.ops4j.pax.logging.log4j2.config.file = ${karaf.etc}/log4j2.config.xml
 
 # Root logger
-log4j.rootLogger=INFO, out, osgi:*
+log4j.rootLogger=INFO, out, osgi:VmLogAppender, osgi:org.codice.ddf.platform.logging.LoggingService
```

Comment:
This change to specify exactly what osgi loggers should receive audit messages was necessary because the new `AdminAlertAppender` was included in `osgi:*`.